prov/verbs/src/fi_verbs.c

```diff
@@ -445,5 +494,15 @@ VERBS_INI
 				 "greater", &verbs_default_inline_size))
 		return NULL;
 
+	if (fi_ibv_get_param_int("min_rnr_timer", "Set min_rnr_timer QP "
+				 "attribute (0 - 31)", &verbs_min_rnr_timer))
+		return NULL;
+
+	if (verbs_min_rnr_timer > 31) {
+		VERBS_WARN(FI_LOG_CORE, "Invalid value for min_rnr_timer."
+			   "Valid range: 0 - 31 \n");
+		return NULL;
+	}
```

Comment:
Yes, but I think the choice is between unexpected behavior (using default value when user had specified a value, albeit erroneous) vs returning an error. I'm leaning towards the latter but I'll let you make the call. :)